docs/features/runs/1_Creating Runs/1_Basic Runs/2_basic-runs.md

```diff
@@ -99,7 +99,7 @@ Once the datastore has been successfully created, you can mount it to a Run usin
 ```text
 git clone https://github.com/PyTorchLightning/grid-tutorials.git
 cd grid-tutorials/features-intro/runs
-grid run --name attaching-datastore --datastore_name cifar5 --datastore_version 1 datastore.py --data_dir /datastores/cifar5/1
```

Comment:
@qugu `--data_dir` is a script parameter, not a Grid parameter. On the contrary, `--datastore_mount_dir` is a Grid parameter which specifies where the data is stored. 

You can see in [the script](https://github.com/Lightning-AI/grid-tutorials/blob/main/features-intro/runs/datastore.py) that it's expecting the param `--data_dir`

Reply:
You are right, sorry!